docs/dev.md

```diff
@@ -0,0 +1,50 @@
+# Dev instructions
+Dev instructions can be found in the [README](https://github.com/nhsengland/swpc_synthea).
+
+# Data Sources 
+## demographics.csv
+This file sets the general demographics for the population created during this simulation, such as age distributions and town populations. The meanings of the different columns in this file can be found [here](https://github.com/synthetichealth/synthea/wiki/Demographics-for-Other-Areas). It can be found at: ```src/main/resources/geography/demographics.csv```.
+
+The demographics breakdowns are done by the full region rather than per town as information per town was not available. 
+
+| Column Name | Contains | Data Sources for UK Version |
+| ----------- | ---------|---------------------------- |
+| ID | Town ID number | monotonically increasing from 1 |
+| COUNTY | code for the county that town is found in | TOWN_211CODE column from [ONS data table](https://www.ons.gov.uk/peoplepopulationandcommunity/populationandmigration/populationestimates/datasets/understandingtownsinenglandandwalespopulationanddemography) |
+| NAME | Town Name | TOWN_211NAME column from above ONS table |
+| STNAME | Region Name | REGION/COUNTRY from above ONS table |
+| POPESTIMATE2015 | Population of that town | TOTAL population rows for 2019 from above ONS data |
+| CTYNAME | County Name where town is located | range of wikipedia sites for the towns |
```

Comment:
Use line 17s language in line 13
COUNTY | Code for county where town is located